recipe/meta.yaml

```diff
@@ -11,20 +11,24 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
     - azure-core >=1.24.0
     - certifi >=2017.4.17
     - isodate >=0.6.0
-    - python >=3.6
+    - python
     - requests >=2.16,<3.dev0
```

Comment:
what's the .dev0 for? Will it work like `<3`?

Reply:
It should. Sometimes you also see 3.0.0a0 to prevent even alphas to be used.